backend/backend/DTOs/RegistrationRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace backend.DTOs
+{
+    public class RegistrationRequest
+    {
+        public string firstname { get; set; }
```

Comment:
would be nice to use PascalCase for property names

Reply:
Done